### PR TITLE
PAGOPA-911-biz-evt-test-gestione-eventi-negativi-per-pdnd

### PR DESCRIPTION
**Pull request by FedericoRuzzier**
#### List of Changes
Adding two scenarios for the integration tests for bizevent: when an insertion happens on the cosmos (awakable or final), a message is sent to the dedicated event hub.
#### Motivation and Context
This addition was necessary to verify that the trigger to the event hub is working correctly. 

#### How Has This Been Tested?
This has been tested through cucumber and the integration test github action

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by FedericoRuzzier** [comment]
update_code

File: .identity/03_github_environment.tf [hunk 1]
```diff
@@ -120,3 +120,12 @@ resource "github_actions_environment_secret" "secret_integration_test_ehub_tx_ne
   plaintext_value  = data.azurerm_key_vault_secret.key_vault_integration_ehub_tx_negative_biz_key[0].value
 }
 
+#tfsec:ignore:github-actions-no-plain-text-action-secrets # not real secret
```
Comment (cap-ang) [inline .identity/03_github_environment.tf]: Maybe this is a typo: it is a real secret.

File: .devops/deploy-pipelines.yml [hunk 2]
```diff
@@ -228,6 +228,7 @@ stages:
                 docker exec -i node_container /bin/bash -c " \
                 cd integration-test
                 export EVENT_HUB_TX_PRIMARY_KEY=${{ variables.eventHubTxPrimaryKey }}
+                export EVENT_HUB_RX_PRIMARY_KEY=${{ variables.eventHubRxPrimaryKey }}
```
Comment (cap-ang) [inline .devops/deploy-pipelines.yml]: A single key was used for the 2 event-hubs? 

There should be two `PRIMARY_KEY`:
- 1 for negative awakable event-hub consumer
- 1 for negative final event-hub consumer

File: .github/workflows/code_review.yml [hunk 3]
```diff
@@ -75,6 +75,7 @@ jobs:
           export CANARY=${{ inputs.canary }}      
           export CUCUMBER_PUBLISH_TOKEN=${{ secrets.CUCUMBER_PUBLISH_TOKEN }}
           export EVENT_HUB_TX_PRIMARY_KEY=${{ secrets.EVENT_HUB_TX_PRIMARY_KEY }}
+          export EVENT_HUB_RX_PRIMARY_KEY=${{ secrets.EVENT_HUB_RX_PRIMARY_KEY }}
```
Comment (cap-ang) [inline .github/workflows/code_review.yml]: as above

File: .identity/03_github_environment.tf (hunk 1 above)
Comment (FedericoRuzzier) [reply .identity/03_github_environment.tf]: Copied from previous secrets, deleting it

File: .github/workflows/integration_test.yml [hunk 4]
```diff
@@ -49,7 +49,10 @@ jobs:
           export CANARY=${{ inputs.canary }}
           export CUCUMBER_PUBLISH_TOKEN=${{ secrets.CUCUMBER_PUBLISH_TOKEN }}
           export EVENT_HUB_TX_PRIMARY_KEY=${{ secrets.EVENT_HUB_TX_PRIMARY_KEY }}
+          export EVENT_HUB_RX_PRIMARY_KEY=${{ secrets.EVENT_HUB_RX_PRIMARY_KEY }}
```
Comment (cap-ang) [inline .github/workflows/integration_test.yml]: useless maybe

**Review by cap-ang — APPROVED** [review]
Good job 👍

**Pull request merged**